config/opensearch_dashboards.yml

```diff
@@ -264,3 +264,7 @@
 
 # Set the value of this setting to false to hide the help menu link to the OpenSearch Dashboards user survey
 # opensearchDashboards.survey.url: "https://survey.opensearch.org"
+
+# Set the value of this setting to false to disable plugin augmentation on Dashboard
```

Comment:
cc @lezzago . I agree with @joshuarrrr , I think we can default it to true?

Reply:
@joshuarrrr @ohltyler I looked at similar settings and they are all set to false and commented out. Maybe those settings were already set to false by default. 

@joshuarrrr if that is the general pattern, then I am fine to have this change made.

Reply:
@ohltyler You can update the yaml file based on the suggested changes. Its good with me.

Reply:
@joshuarrrr can you suggest a change to change `false` to `true` on line 269 as well? It is easier than doing a direct commit to the feature branch since we are only doing that via PRs

Reply:
Or I can add the change in a separate PR but that doesn't seem necessary

Reply:
Nvm I will fix in separate PR with a UT fix.